core-bundle/contao/dca/tl_image_size.php

```diff
@@ -328,9 +328,11 @@ public function getFormats(DataContainer|null $dc=null)
 			$formats = StringUtil::deserialize($dc->value, true);
 		}
 
+		$imageExtensions = System::getContainer()->getParameter('contao.image.valid_extensions');
+
 		foreach ($this->getSupportedFormats() as $format => $isSupported)
 		{
-			if (!in_array($format, System::getContainer()->getParameter('contao.image.valid_extensions')))
+			if (!in_array($format, $imageExtensions))
```

Comment:
This change seems to be unrelated to this pull request.

Reply:
It's unrelated to this PR and was done almost 6 months ago (you could see it in the history).

Unsure about the performance but executing this for every supported format can be saved by just doing getting it once.

Reply:
See: https://github.com/contao/contao/pull/7506